src/Rector/Deprecation/UiHelperTraitDrupalPostFormRector.php

```diff
@@ -12,11 +12,6 @@
 final class UiHelperTraitDrupalPostFormRector extends AbstractRector
 {
 
-    /**
-     * @todo remove when property is no longer private in AbstractRector.
-     */
-    private $nodesToAddCollector;
-
```

Comment:
Keep but make protected since change is not released.

Reply:
Done